tests/common/test_image.py

```diff
@@ -8,28 +8,108 @@
 import numpy as np
 from gtsam import Cal3Bundler
 
-from gtsfm.common.image import Image
+from gtsfm.common.image import Image, DEFAULT_FOCAL_LENGTH_FACTOR
 from gtsfm.common.sensor_width_database import SensorWidthDatabase
 
 
 class TestImage(unittest.TestCase):
     """Unit tests for the image class."""
 
+    def test_get_intrinsics_from_exif_no_exif(self):
+        """
+        Tests the intrinsics generation from exif.
+        Focal length default value is expected because exif data is None.
+        """
+
+        im_h = 100
+        im_w = 120
+        expected_focal_length = DEFAULT_FOCAL_LENGTH_FACTOR * max(im_h, im_w)
+        expected_instrinsics = Cal3Bundler(fx=expected_focal_length, k1=0.0, k2=0.0, u0=60.0, v0=50.0)
```

Comment:
nit: I would recommend putting the comment "Focal length default value is expected because exif data is None." inline where we check this, and then making the 1-line docstring fit on a single line as

```python
"""Tests the intrinsics generation from exif."""
```
I would also put the `expected_focal_length` line next to where it is used, right above line 31

Reply:
Marking as resolved, thanks!